.stylelintrc

```diff
@@ -1,6 +1,8 @@
 {
     "extends": ["@gravity-ui/stylelint-config", "@gravity-ui/stylelint-config/prettier"],
+    "plugins": ["stylelint-use-logical"],
     "rules": {
+        "csstools/use-logical": ["always", {"except": ["float", "width", "min-width", "max-width", "height", "min-height", "max-height"]}],
```

Comment:
why this props are exception?

Reply:
float has weak browsers support, widht and height needed for different `writing-mode` which we are not planning to support now